SurveyApp/Modules/Home/SurveyCollectionCell/SurveyCollectionCell.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  SurveyCollectionCell.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 6/21/21.
+//
+
+import UIKit
+import SnapKit
+
+final class SurveyCollectionCell: UICollectionViewCell {
+    
+    private let imageView = UIImageView()
+    private let titleLabel = UILabel()
+    private let descriptionLabel = UILabel()
+    private let gradientLayer = CAGradientLayer()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setUpLayout()
+        setUpViews()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func prepareForReuse() {
+        super.prepareForReuse()
+    }
+    
+    func setTitle(_ title: String) {
+        titleLabel.text = title
+    }
+    
+    func setDescription(_ description: String) {
+        descriptionLabel.text = description
+    }
+    
+    func setImage(imageURLString: String) {
+        guard let imageURL: URL = URL(string: imageURLString) else { return }
+        imageView.loadUrl(url: imageURL)
+    }
+    
+    // TODO: might not need when integrate
+    func setImage(image: UIImage) {
+        imageView.image = image
+    }
```

Comment:
I think we can group these into one function, such: `func configure(with title:String...)`, cause most of the time these ones will be set all at once, IMO 🤔

Reply:
Thank you for better approach 🙇‍♀️